.gitlab-ci.yml

```diff
@@ -160,6 +160,8 @@ test-apm-injection:
 
 test-observability-pipelines-worker:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
+  rules:
+  - if: '$CI_PIPELINE_SOURCE == "push"'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Was this for testing or intended to be left in? If intentional, why is it the only job with this rule?

Reply:
It was intended to be here, so this job is not triggered when this pipeline is triggered from `datadog-agent` repository.